server/tests/api/test_rounds.py

```diff
@@ -137,6 +198,60 @@ def test_rounds_create_two(
     assert sorted(sampled_jurisdictions) == sorted(jurisdiction_ids[:2])
 
 
+def test_multiwinner_rounds_create_two(
+    client: FlaskClient,
+    election_id: str,
+    jurisdiction_ids: List[str],
+    mw_contest_ids: List[str],
+    mw_round_1_id: str,
+    snapshot,
+):
+    run_mw_audit_round(mw_round_1_id, mw_contest_ids[0], mw_contest_ids, 0.5, 0.3)
+
+    rv = post_json(client, f"/api/election/{election_id}/round", {"roundNum": 2},)
+    assert_ok(rv)
+
+    expected_rounds = {
+        "rounds": [
+            {
+                "id": assert_is_id,
+                "roundNum": 1,
+                "startedAt": assert_is_date,
+                "endedAt": assert_is_date,
+                "isAuditComplete": False,
+            },
+            {
+                "id": assert_is_id,
+                "roundNum": 2,
+                "startedAt": assert_is_date,
+                "endedAt": None,
+                "isAuditComplete": None,
+            },
+        ]
+    }
+    rv = client.get(f"/api/election/{election_id}/round")
+    rounds = json.loads(rv.data)
+    compare_json(rounds, expected_rounds)
+
+    set_logged_in_user(client, UserType.JURISDICTION_ADMIN, DEFAULT_JA_EMAIL)
+    rv = client.get(
+        f"/api/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/round"
+    )
+    rounds = json.loads(rv.data)
+    compare_json(rounds, expected_rounds)
+
+    ballot_draws = SampledBallotDraw.query.filter_by(
+        round_id=rounds["rounds"][1]["id"]
+    ).all()
+    # Check that we automatically select the 90% prob sample size
```

Comment:
i thought the point here was to make sure we were falling back to the `asn` - is this comment just out of date?

in general, if the only thing we need to test is that the `asn` sample size gets picked for round 2 in the multiwinner case, i think this test case alone should suffice.

do you think there are other important things to test about multiwinner that made you want to add more test cases? (i'm not opposed to more tests, just want to understand what they are testing)

Reply:
i see that in multi-winner we are also expecting a different response for the first round sample sizes than in single-winner, which makes me think we should also have a test that confirms the sample size options that are returned in the first round